### ISSUE TITLE
yarn scripts use npm

### ISSUE THREAD

**Issue by penx** [bahmutov/start-server-and-test#255]
## Is this a bug report or a feature request?

If this is a bug report, please provide as much info as possible

- version 1.11.0
- platform Mac OS X
- expected behavior

With the following:

```json
    "scripts": {
        "start-server": "yarn start",
        "test": "mocha e2e-spec.js",
        "ci": "start-server-and-test start-server http://localhost:8080 test"
    }
```

Running `yarn ci` should run `yarn start-server` followed by `yarn test`

- actual behavior

Uses yarn rather than npm:

```
1: starting server using command "npm run start-server"
and when url "[ 'http://localhost:8080' ]" is responding with HTTP status code 200
running tests using command "npm run test"
```

If this can't be fixed soon, let me know and I can open a PR to add this workaround in to the docs:

```json
    "scripts": {
        "start-server": "npm start",
        "test": "mocha e2e-spec.js",
        "ci": "start-server-and-test 'yarn start-server' http://localhost:8080 'yarn test'"
    }
```

**Comment by bahmutov** [comment]
Yeah i won’t have time to get to this, so PR to at least mention the workaround would be nice

**Comment by detj** [comment]
Should there be a configuration that hints `start-server-and-test` to use `yarn` instead of `npm` by default?

**Comment by Lyokolux** [comment]
I don't have free time too. Can you add @bahmutov a label such as "PR welcome", "Help wanted", etc... to make it more visible ?

**Comment by Lyokolux** [comment]
> Should there be a configuration that hints `start-server-and-test` to use `yarn` instead of `npm` by default?

I don't have free time too. Can you add @bahmutov a label such as "PR welcome", "Help wanted", etc... to make it more visible ?
